### PR TITLE
v1.4.2 (2024 Tree Cover Loss Update)

### PR DESCRIPTION
**Pull request by melrose-wri**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [x] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?
The model currently uses tree cover loss, tree cover loss from fires, and 1km drivers of tree cover loss through 2023, and do not include N2O emissions from soil nitrogen mineralization.

Issue Number: N/A

## What is the new behavior?
- Updated model to calculate fluxes through 2024 using new tree cover loss, tree cover loss from fires, and drivers of tree cover loss which have all been updated through 2024.
- Included N2O emissions from soil nitrogen mineralization in emissions calculation step.
- Updated the "hansenize" pre-processing step to use only band1 for the 1km drivers of tree cover loss.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ The framework essentially spatially applies IPCC national greenhouse gas invento
 It covers only forests converted to non-forests, non-forests converted to forests and forests remaining forests (no other land 
 use transitions). The framework is described and published in [Harris et al. (2021) Nature Climate Change
 "Global maps of twenty-first century forest carbon fluxes"](https://www.nature.com/articles/s41558-020-00976-6).
```
Comment (dagibbs22) [inline readme.md:14]: "The framework is described and published in [Harris et al. (2021) Nature Climate Change
"Global maps of twenty-first century forest carbon fluxes"](https://www.nature.com/articles/s41558-020-00976-6) and was updated in [Gibbs et al. (2025) "Revised and updated geospatial monitoring of 21st century forest carbon fluxes"](https://essd.copernicus.org/articles/17/1217/2025/).

File: readme.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ The framework essentially spatially applies IPCC national greenhouse gas invento
...
-Although the original manuscript covered 2001-2019, the same methods were used to update the framework to include 2023, 
+Although the original manuscript covered 2001-2019, the same methods were used to update the framework through 2024, 
 with a few changes to some input layers and constants. You can read about the changes since publication 
-[here](https://www.globalforestwatch.org/blog/data-and-research/whats-new-carbon-flux-monitoring).
+[here](https://www.globalforestwatch.org/blog/data-and-tools/whats-new-carbon-flux-monitoring/).
```
Comment (dagibbs22) [inline readme.md:17]: "Updates have been made to the framework since the original publication, which covered 2001-2019. The most up-to-date description of model changes is available [here](https://www.globalforestwatch.org/blog/data-and-tools/whats-new-carbon-flux-monitoring/)."

File: readme.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ A complete list of inputs, including changes made to the framework since the ori
 [here](http://gfw2-data.s3.amazonaws.com/climate/carbon_model/Table_S3_data_sources__updated_20230406.pdf).
```
Comment (dagibbs22) [inline readme.md:39]: Delete "A complete list of inputs, including changes made to the framework since the original publication, can be found 
[here](http://gfw2-data.s3.amazonaws.com/climate/carbon_model/Table_S3_data_sources__updated_20230406.pdf).". I'm not updating this anymore.

**Review by dagibbs22 — CHANGES_REQUESTED** [review]
I'm requesting a few changes to the readme and other comments.

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  given-names: "David"
+  orcid: "https://orcid.org/0000-0001-7389-8569"
+- family-names: "Rose"
+  given-names: "Melissa"
+  orcid: "https://orcid.org/0000-0003-0012-0734"
+- family-names: "Harris"
+  given-names: "Nancy"
+  orcid: "https://orcid.org/0000-0001-6661-4249"
+title: "GFW Forest Carbon Flux Model"
+version: 1.4.1
```
[the remark this answers is not in the document]
Comment (melrose-wri) [reply CITATION.cff]: Resolved this here: https://github.com/wri/carbon-budget/pull/61/commits/314ab063b68bc05e15cd7364485c7acc09224e02

File: constants_and_names.py [hunk 5]
```diff
@@ -206,14 +206,14 @@
...
-loss_dir = 's3://gfw2-data/forest_change/hansen_2023/'
+pattern_loss = 'GFW2024'
+loss_dir = 's3://gfw2-data/forest_change/hansen_2024/'
 
 # Hansen removals tiles based on canopy height (2000-2020)
 # From https://www.frontiersin.org/articles/10.3389/frsen.2022.856903/full
 pattern_data_lake = ''
 pattern_gain_ec2 = 'tree_cover_gain_2000_2020'
-gain_dir = 's3://gfw-data-lake/umd_tree_cover_gain_from_height/v202206/raster/epsg-4326/10/40000/gain/geotiff/'
+gain_dir = 's3://gfw-data-lake/umd_tree_cover_gain_from_height/v20240126/raster/epsg-4326/10/40000/is/geotiff/'
```
[the remark this answers is not in the document]
Comment (melrose-wri) [reply constants_and_names.py]: Resolved this here: https://github.com/wri/carbon-budget/pull/61/commits/79232dbb3d7a794dd325aa4113e3f48f4da68452

File: readme.md (hunk 2 above)
Comment (melrose-wri) [reply readme.md]: Resolved this here: https://github.com/wri/carbon-budget/pull/61/commits/52ceddf4b8a6b21605036494abc3ff81470aed0e
